index.html

```diff
@@ -62,15 +60,15 @@ <h1 class="invite_phrase">
   </section>
 
 
-  <section class="section second_page_container">
+  <section class="section second_page_container section_pink_wrapper">
     <h2 class="section_heading">
       We offer a handpicked range of cabins for nature lovers in Europe to live and work
       from for desired time.
     </h2>
-    <div class="flex-j-center">
+    <div class="flex-j-center flex_column">
       <div class="pros_container">
```

Comment:
instead of setting margin on pros_container you should set padding to section container. So you do it once and control every layout spacing in every section

Reply:
so fix it for every section 😉

Reply:
fixed only last section to go but I need to check a code for the last section to be ready to change that

Reply:
fixed i guess...